_studio/shared/umc/codec/av1_dec/src/umc_av1_decoder_va.cpp

```diff
@@ -89,6 +89,8 @@ namespace UMC_AV1_DECODER
             packer->EndFrame();
 
         sts = va->Execute();
+        if (sts != UMC::UMC_OK)
```

Comment:
aha. So, here it is where we missed an error from media driver. Ok, that's a gap. However, why do you change the status here instead of just 'return sts'? /which status we actually get from Execute?/ And  you could just return UMC::UMC_ERR_DEVICE_FAILED to avoid one more if in the above code...

Reply:
We receive UMC::UMC_ERR_INVALID_PARAMS from driver

Reply:
UMC::UMC_ERR_DEVICE_FAILED is located here:
https://github.com/Intel-Media-SDK/MediaSDK/blob/07d6b5f50a2839f570bfb856179083d7592283fd/_studio/shared/umc/core/umc/include/umc_va_base.h#L182-L187
And it is used in switch-case in mfx_av1_dec_decode.cpp under `#if defined(MFX_VA)`

Reply:
@onabiull : how about promoting these 3 guys to normal UMC status codes?

@aobolensk : the whole this file seems to be VA specific. Can we just use UMC::UMC_ERR_DEVICE_FAILED here?

Reply:
Replaced UMC::UMC_ERR_FAILED with UMC::UMC_ERR_DEVICE_FAILED